config/crd/ocm/addon.open-cluster-management.io_clustermanagementaddons.crd.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
Why do we need these CRDs?  These are common and I'd expect the add-on framework to install these.
If each add-on tries to install these, it will create problems, especially if upgrading or deleting the helm release.

Reply:
these CRDs are install by OCM, in e2e tests,  OCM is not deployed, so need to install them for e2e tests.

Reply:
they are deployed here in run-search-operator-e2e.sh https://github.com/stolostron/search-operator/pull/99/files#diff-ff7c1281606eabd00ba3bbc58b1bef5e6c121164394b948c7371071e684f100dR144 for e2e tests.

Reply:
Thanks! I thought these were going to the managed cluster.  It's good if it's only for the tests.